src/neuroconv/datainterfaces/ophys/tdt_fp/tdtfiberphotometrydatainterface.py

```diff
@@ -157,39 +174,50 @@ def add_to_nwbfile(
             description=metadata["Ophys"]["FiberPhotometry"]["FiberPhotometryTable"]["description"],
         )
         for row_metadata in metadata["Ophys"]["FiberPhotometry"]["FiberPhotometryTable"]["rows"]:
-            fiber_photometry_table.add_row(
```

Comment:
What is going on with this change?

Reply:
Not all of these fields are optional.
Optional Fields: coordinates, notes, commanded_voltage_series, emission_filter, and excitation_filter.
Required Fields: location, indicator, optical_fiber, excitation_source, photodetector, dichroic_mirror.

Reply:
I moved the required fields outside of the if statements @pauladkisson